server/pxf-jdbc/src/main/java/org/greenplum/pxf/plugins/jdbc/JdbcBasePlugin.java

```diff
@@ -117,34 +129,53 @@ public void initialize(RequestContext context) {
         if (quoteColumnsRaw != null) {
             quoteColumns = Boolean.parseBoolean(quoteColumnsRaw);
         }
+
+        // This parameter is not required. The default value is empty map
+        Map<String, String> sessionConfigurationRaw = configuration.getPropsWithPrefix(JDBC_SESSCONF_PROPERTY_PREFIX);
+        sessionConfiguration.putAll(sessionConfigurationRaw);
+        LOG.debug("Session configuration: {}",
+            sessionConfiguration.entrySet().stream()
+                    .map(entry -> "'" + entry.getKey().toString() + "'='" + entry.getValue() + "'")
+                    .collect(Collectors.joining(", "))
+        );
+
+        // This parameter is not required. The default value is empty map
+        Map<String, String> connectionConfigurationRaw = configuration.getPropsWithPrefix(JDBC_CONNCONF_PROPERTY_PREFIX);
+        connectionConfiguration.putAll(connectionConfigurationRaw);

```

Comment:
2 lines above can be combined as connectionConfigurationRaw variable is never referred to anywhere else.
```suggestion
        connectionConfiguration.putAll(configuration.getPropsWithPrefix(JDBC_CONNECTION_PROPERTY_PREFIX));
```